src/packages/cli/package.json

```diff
@@ -138,7 +138,7 @@
     "test:postgresql": "jest integration.postgresql --maxConcurrency=10",
     "test:mysql": "jest integration.mysql --maxConcurrency=10",
     "test:mariadb": "jest integration.mariadb --maxConcurrency=10",
-    "test:commands": "./fixtures/test.sh && jest dotenv- format doctor generate studio && jest tests__/introspect",
+    "test:commands": "./fixtures/test.sh && jest dotenv- format doctor generate && jest tests__/introspect && jest studio --maxConcurrency=1",
```

Comment:
Did some digging, and it turns out concurrency doesn't play well with Studio tests yet because it tries to generate multiple Prisma Clients at the same time. I set its concurrency to 1 and moved it to a separate command so other tests don't have to pay this penalty